draft-ietf-httpapi-ratelimit-headers.md

```diff
@@ -186,12 +186,12 @@ The goals do not include:
 {::boilerplate bcp14}
 
 This document uses the Augmented BNF defined in {{!RFC5234}} and updated
-by {{!RFC7405}} along with the "#rule" extension defined in Section 7 of
-{{!MESSAGING=RFC7230}}.
+by {{!RFC7405}} along with the "#rule" extension
+defined {{!MESSAGING=RFC7230}}, Section 7.
```

Comment:
```suggestion
defined in Section 7 of {{!MESSAGING=RFC7230}}, .
```

Reply:
I'm using Martin's i-d-template docker image, so I'll switch to "in Section xxx of yyy"